src/main.cpp

```diff
@@ -203,16 +228,73 @@ void rpcWSWrapper() {
 void GetDeviceInfo() {
   JsonDocument jsonResponse;
   jsonResponse["name"] = shelly_name;
-  jsonResponse["id"] = shelly_name;
+  jsonResponse["id"] = shelly_name; // ???? ToDo check
   jsonResponse["mac"] = shelly_mac;
-  jsonResponse["slot"] = 1;
-  jsonResponse["model"] = "SPEM-003CEBEU";
-  jsonResponse["gen"] = 2;
-  jsonResponse["fw_id"] = "20241011-114455/1.4.4-g6d2a586";
-  jsonResponse["ver"] = "1.4.4";
-  jsonResponse["app"] = "Pro3EM";
+  jsonResponse["slot"] = shelly_slot;
+  jsonResponse["model"] = shelly_model;
+  jsonResponse["gen"] = shelly_gen;
+  jsonResponse["fw_id"] = shelly_fw_id;
+  jsonResponse["ver"] = shelly_ver;
+  jsonResponse["app"] = shelly_app;
   jsonResponse["auth_en"] = false;
-  jsonResponse["profile"] = "triphase";
+  jsonResponse["auth_domain"] = nullptr;
+  jsonResponse["profile"] = power_variant;

```

Comment:
würde ich gerne auf triphase belassen, da dies von hoymiles gefordert wird. trotzdem kann das device ja die http endpoints für monophase zur verfügung stellen.

Reply:
Wäre es ein Kompromiss, den Standard auf "triphase" zu stellen aber an dieser Stelle es nicht fest zu verankern? Grund hierfür ist auch wieder die Auswertung des Gerätes in FHEM Bsp. weise. Sobald ich den Wert fest schreibe, gibt es Konflikte in der Auswertung.

Reply:
Wieso gibt es Konflikte? Der echte shelly könnte doch triphase konfiguriert sein sodass fhem das auch akzeptieren muss, oder verstehe ich das falsch?

Reply:
s.o.